app/scripts/lib/email-verification-reasons.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * List of reasons to verify emails
+ */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  return {
+    ACCOUNT_UNLOCK: 'account_unlock',
+    PASSWORD_RESET: 'password_reset',
+    SIGN_IN: 'sign_in',
```

Comment:
> Maybe this file should be called `email-challenge-reasons` to go along with the nomenclature that will be returned by the auth server. And if we are doing that, we should use the same values for consistency and simplicity.

Also, although maybe I'm missing some context, is it worth dropping the `email-` part in expectation of different challenge types landing in the future?